CHANGELOG.md

```diff
@@ -25,10 +43,6 @@ These should not affect the functionality of the module.
 
 [Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/3.9.0...v4.0.0)
 
-This is the first release in VoxPupuli's `puppet` namespace.
```

Comment:
What happened to this release summary?

Reply:
Dunno.. just ran the github_changelog_generator

Reply:
With the github version from the Gemfile?
Release summaries is quite a new feature and unfortunately there hasn’t been an actual release of gcg for a long time now.